Integrations/integration-IsItPhishing.yml

```diff
@@ -0,0 +1,166 @@
+description: Collaborative web service that provides validation on whether a URL is
+  a phishing page or not by analyzing the content of the webpage
+configuration:
+- display: Customer's name
+  name: name
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Customer's License
+  name: license
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Server URL
+  name: url
+  defaultvalue: https://ws.isitphishing.org
+  type: 0
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |+
+    var auth = 'Bearer ' + Base64.encode(params.name + ':' + params.license);
+
+    var sendRequest = function(method, api, body) {
+        var url = params.url;
+        if (url[url.length - 1] === '/') {
```

Comment:
while will take care of more than one / at the end of the string.
you can also use regex to eliminate them